### PR TITLE
chore: upgrade deps and remove type hack from widget

### PR DESCRIPTION
**Pull request by domoritz**
Resolves a typing issue: https://twitter.com/maartenbreddels/status/1638204921701883905.

### CODE REVIEW

File: packages/duckdb/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "prepublishOnly": "npm run test && npm run lint"
   },
   "dependencies": {
+    "@mapbox/node-pre-gyp": "^1.0.10",
```
Comment (jheer) [inline packages/duckdb/package.json]: Why is this needed?

Comment (domoritz) [reply packages/duckdb/package.json]: Let's try again but the CI build for duckdb failed without it when I tried two weeks ago.

Comment (domoritz) [reply packages/duckdb/package.json]: Seems to still work even when I remove the explicit dependency now.
